ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -3592,7 +3592,7 @@ private void SubmitStopLossInternal()
             if (tracing)
                 Print("submitStopLossInternal(" + orderEntry.ToString() + ")");
 
-            if (IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode)

```

Comment:
This is required - IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode use the same paths a lot of the time-